patches/octane_run_we.js

```diff
@@ -0,0 +1,47 @@
+load('base.js');
+load('richards.js');
+load('deltablue.js');
+load('crypto.js');
+load('raytrace.js');
+load('earley-boyer.js');
+load('regexp.js');
+load('splay.js');
+load('navier-stokes.js');
+load('pdfjs.js');
+//load('mandreel.js');
+//load('gbemu-part1.js');
+//load('gbemu-part2.js');
+//load('code-load.js');
+//load('box2d.js');
+//load('zlib.js');
+//load('zlib-data.js');
+load('typescript.js');
+load('typescript-input.js');
+load('typescript-compiler.js');
+
+
+function main() {
+  var suites = BenchmarkSuite.suites;
+  for (var i = 0; i < suites.length; i++) {
+    var suite = suites[i];
+    var benchmarks = suite.benchmarks;
+    for (var j = 0; j < benchmarks.length; j++) {
+      var benchmark = benchmarks[j];
+      print(benchmark.name);
+      for (var k = 0; k < 2000; k++) {
```

Comment:
Can you add comments saying what each loop does here? Presumably this innermost one is in-process iterations? Which would mean the next ourmost loop should be process executions, but it can't be, as we are in the same process still...